examples/text-classification/run_glue.py

```diff
@@ -133,12 +132,51 @@ class ModelArguments:
     )
 
 
+# Should be moved to transformers/src/transformers/file_utils.py
+def is_run_on_sagemaker():
+    if "SM_OUTPUT_DATA_DIR" in os.environ and "SM_MODEL_DIR" in os.environ:
+        return True
+    else:
+        return False
+
+
+# Should be moved to transformers/src/transformers/file_utils.py or transformers/src/transformers/hf_argparser.py
+def parse_sagemaker_args(argv):
```

Comment:
I feel like `parse_sagemaker_args ` might not be the right name here. Maybe `parse_sagemaker_env_into_args`?
or `inject_sagemaker_env_into_args`?

What do others think?

Reply:
I like `parse_sagemaker_env_into_args`.